app/ui/src/app/customizations/tech-extensions/tech-extensions-list.component.html

```diff
@@ -48,7 +48,7 @@ <h1>Technical Extensions</h1>
                      let-index="index">
             <button type="button"
                     class="btn btn-default"
-                    [routerLink]="['import', item.id, { 'name': item.name }]">Update</button>
+                    [routerLink]="['import', item.id, item.extensionId]" [queryParams]="{ name: item.name }">Update</button>
```

Comment:
Ok, there's still one issue left. For the current implementation `Update` button from list page works, because of this change. But there is another `Update` button on Detail page that isn't routed properly now.

https://github.com/syndesisio/syndesis/blob/98ff655cfa8e50aaa049d1f0c2d70f27d5e1ff08/app/ui/src/app/customizations/tech-extensions/detail/tech-extension-detail.component.html#L40